src/Workspaces/Core/Desktop/Workspace/FileTextLoader.cs

```diff
@@ -72,7 +72,6 @@ protected virtual SourceText CreateText(Stream stream, Workspace workspace)
             TextAndVersion textAndVersion;
             using (var stream = FileUtilities.OpenAsyncRead(_path))
             {
```

Comment:
@davkean did you mean the `TempStream` type?  If so then no it does not.  The `TempStream` type is used for a very narrow scenario in signing.